validation_pipeline.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.      predictions.dropna(subset=['fraud', 'fraud_p_value'], inplace=True)
</pre><p>why do we drop these?  if we get a nan p-value, is something is wrong with the input set?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
No, it just means that we don't have enough samples for that grouping (i.e. just one measurement for the element). When it's a combined p value (Carbon, Oxygen and Nitrogen). It means that no element had enough measurements (>= 1)

Reply:
<p>Ok, now I'm confused. Can we define "grouping" somewhere? Does this refer to measurements from an individual sample?</p>

Reply:
Yes, the grouping here is [code, long, lat] which is what we define as a group of samples.